test/integration/action_execution.go

```diff
@@ -192,20 +202,29 @@ var _ = Describe("Action Executor ", func() {
 
 	// Test resize action execution
 	Describe("executing resize action on a deployment", func() {
-		It("should match the expected resource cpu reuqest after resize", func() {
+		It("should match the expected resource request/limit after resizing on both of cpu and memory", func() {
 
 			dep, err := createDeployResource(kubeClient, depSingleContainerWithResources(namespace, "", 1, false, false, false))
 			framework.ExpectNoError(err, "Error creating test resources")
 
 			targetSE := newResizeWorkloadControllerTargetSE(dep)
-			currentSE := newContainerEntity("test-cont")
 
-			_, err = actionHandler.ExecuteAction(newResizeActionExecutionDTO(proto.ActionItemDTO_RIGHT_SIZE, targetSE, nil, currentSE), nil, &mockProgressTrack{})
-			framework.ExpectNoError(err, "Resize action failed")
+			// Resize on limit, increasing cpu/memory limit by 100m/100Mi
+			_, err = actionHandler.ExecuteAction(newResizeActionExecutionDTO(proto.ActionItemDTO_RIGHT_SIZE, targetSE, LIMIT_SINGLE_CONTAINER), nil, &mockProgressTrack{})
+			framework.ExpectNoError(err, "Resize action on limit failed")
 
-			_, err = waitForDeploymentToUpdateResource(kubeClient, dep)
+			_, err = waitForDeploymentToUpdateResource(kubeClient, dep, LIMIT_SINGLE_CONTAINER)
 			if err != nil {
-				framework.Failf("Failed to check the resource request in the new deployment: %s", err)
+				framework.Failf("Failed to check the change of resource/limit in the new deployment: %s", err)
+			}
+
+			// Resize on request, increasing cpu/memory request by 100m/100Mi
```

Comment:
The comment here should not be specific to the actual test.

Reply:
done